From 1f259bc12b4de089de7eb5e2704d0b51227f7cb1 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sat, 28 May 2022 23:53:09 +0300
Subject: [PATCH] [SW-0] greco: raise arm preboot size to fit code

Raise by 1KB, so the new code can fit the .sram section.

Change-Id: Iaf3283bb29ee38b24746568414704834549e4448
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/greco/arm/common.h b/arch/greco/arm/common.h
index 6c65aee..a9c93e1 100644
--- a/arch/greco/arm/common.h
+++ b/arch/greco/arm/common.h
@@ -11,7 +11,7 @@
 #include <sizes.h>
 
 #define PPBOOT_SIZE			0x5000		/* 20KB */
-#define PREBOOT_SIZE			0x12800		/* 74KB */
+#define PREBOOT_SIZE			0x12C00		/* 75KB */
 
 #define HL_SP_SRAM_SIZE			SZ_64K
 
